corehq/ex-submodules/couchexport/export.py

```diff
@@ -309,7 +309,8 @@ def split_path(path):
         id = []
         for k in path:
             if isinstance(k, basestring):
-                column.append(k)
+                if k:
+                    column.append(k)
```

Comment:
yes, that's correct, if you have any group and then remove it and add a question with the same id the schema will get an empty `''` key.

If you then want to export the plain question (and not the deleted group) you are unable to since 'key' for the column is `question_id.` (note trailing dot).

This just removes the empty key from the path. It should affect anything if you want to export the deleted group since the path to that will not contain an empty key.